backend/src/main/java/de/neuefische/henny/reisekasse/Model/EventMember.java

```diff
@@ -0,0 +1,17 @@
+package de.neuefische.henny.reisekasse.Model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class EventMember {
+
+    private String username;
+    private double balance;
```

Comment:
😱  Don't use `float` or `double` to represent money! Better count it in cents and store in an `int` or `long`. Maybe this is overkill for your purpose, but Java has a `BigDecimal` to store integral numbers of arbitrary precision.

For a good explanation see the prefered answer at [Why not use Double or Float to represent currency?](https://stackoverflow.com/a/3730040)